knack/log.py

```diff
@@ -75,10 +59,7 @@ def __init__(self, log_level_config, log_format, enable_color):
     def format(self, record):
         msg = logging.StreamHandler.format(self, record)
         if self.enable_color:
-            try:
-                msg = self.get_color_wrapper(record.levelno)(msg)
-            except KeyError:
-                pass
+            msg = self.wrap_with_color(record.levelname, msg)
```

Comment:
We shouldn't ignore `KeyError`. As `KeyError` indicates `color_map` is corrupted.